docs/conceptual/Bar-Memory.rst

```diff
@@ -0,0 +1,98 @@
+.. meta::
+   :description: Learn about BAR memory and ways to handle physical addressing limit in ROCm
+   :keywords: BAR memory, MMIO, GPU memory, Physical Addressing Limit, AMD, ROCm
+
+******************************
+BAR memory overview
+******************************
+From the BIOS of an Intel Xeon E5-based system, you can enable above 4 GB PCIe addressing. You also need to set the
```

Comment:
The intro sounds like it's mid-conversation. It also doesn't mention BAR memory at all. I would change it to something that includes BAR memory and introduces the topic. I would pull the info from later up and put it up here:

"P2P DMA only works when one device can directly access the local BAR memory of another. If the BAR memory is above the physical addressing limit of the device, it will not be able to access the remote BAR."

and then say what ways are used and move on from there.